src/pytest_xray/evidence.py

```diff
@@ -50,3 +53,72 @@ def json(data: AnyStr, filename: str) -> Dict[str, str]:
 
 def zip(data: AnyStr, filename: str) -> Dict[str, str]:
     return evidence(data, filename, APP_ZIP)
+
+
+@pytest.fixture
+def xray_evidence(request) -> Callable:
+    """ Fixture to add an evidence to the Test Run details of a Test.
+
+    Copyright © 2023 Orange - All rights reserved
```

Comment:
This repo is under Apache license. I am not an expert for licenses, that why I would like to avoid having some copyrights that may not be compatible with Apache license.